test/unit/widgets/buttons.test.ts

```diff
@@ -179,6 +191,54 @@ describe('continueButton', () => {
   })
 })
 
+describe('createLinkDiv', () => {
+  const obj = namedNode('https://test.com/#name')
+  const options: RenderAsDivOptions = {
+    deleteFunction: () => {},
+    link: true
+  }
+
+  it('adds a div to the element provided', () => {
+    createLinkDiv(dom, element, obj, options)
+    expect(element.children.length).toBeGreaterThan(0)
+  })
+  it('makes the element draggable', () => {
+    createLinkDiv(dom, element, obj, options)
+    expect(element.getAttribute('draggable')).toEqual('true')
+  })
+  it('adds the style....', () => {
+    createLinkDiv(dom, element, obj, options)
+    expect(element.children[0].getAttribute('style')).toEqual(linkDivStyle)
+  })
+  it('adds the deleteFunction of .... deleteButton with Check', () => {
+    const options = {
+      deleteFunction: () => {}
+    }
+    createLinkDiv(dom, element, obj, options)
+  })
+  it('adds the link icon and link for the uri if link option is true', () => {
+    const options = {
+      link: true
+    }
+    createLinkDiv(dom, element, obj, options)
+    expect(element.children[0].children[0].nodeName).toEqual('A')
```

Comment:
**thought** maybe testing for correct element.innerHtml would fit better :thinking: We should also think about introducing [testing-library/dom](https://testing-library.com/docs/dom-testing-library/example-intro), like I did in solid-panes. Not necessarily in this PR, but perhaps you can take a look and tell what you think about it.

Reply:
I have installed testing library. I tried to use it here and I had some trouble.  I'm wondering if you give me an example with this one and I can do it to the other test I created like this as well.